scripts/triggered_mails.py

```diff
@@ -0,0 +1,42 @@
+import sys
+import logging
+from datetime import datetime, timedelta
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+from website.app import init_app
+from website import mails, settings
+from framework.auth import User
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+def main(dry_run=True):
+    #one query for 6 weeks and osf4m users, and one for 4 weeks for regular users
+    inactive_users = list(User.find(Q('date_last_login', 'lt', datetime.utcnow() - settings.NO_LOGIN_WAIT_TIME) &
+                                    Q('conference_user', 'eq', False)))
+    inactive_users.extend(list(User.find(Q('date_last_login', 'lt', datetime.utcnow() - settings.NO_LOGIN_OSF4M_WAIT_TIME) &
+                                    Q('conference_user', 'eq', True))))
+    inactive_emails = list(mails.QueuedMail.find(Q('email_type', 'eq', 'no_login')))
+    users_sent = []
+    for email in inactive_emails:
```

Comment:
You can use a list comprehension here instead: `users_sent = [email.user for email in inactive_emails]`